js/gui.js

```diff
@@ -47,6 +47,32 @@ var keyHandlers = {
   },
 };
 
+// On Screen Keys
+$("#leftKey").on("click",function(){
```

Comment:
If you put this into the document onload, you can add the scripts to the top.

Reply:
This thing or the entire script?

Reply:
the entire script. Then, it can be included in the head.

Reply:
On it